projects/libavc/project.yaml

```diff
@@ -0,0 +1,11 @@
+homepage: "https://android.googlesource.com/platform/external/libavc/"
+primary_contact: "harish.mahendrakar@ittiam.com"
+sanitizers:
+- address
```

Comment:
can you please enable memory and undefined later as well. memorysanitizer should work since it seems self contained and does not seem to have system deps (when i see dockerfile).

Reply:
Yes. Waiting for few CLs to be merged to aosp/master on libavc. Will enable both, once aosp/master is updated.